Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -196,6 +197,54 @@ class SettingsViewController: TableViewController {
                     self.navigationController?.pushViewController(clearPrivateData, animated: true)
                 },
                 accessory: .disclosureIndicator
+            ),
+            Row(
+                text: Strings.Block_all_cookies,
+                accessory: .switchToggle(
+                    value: Preferences.Privacy.blockAllCookies.value, { [unowned self] in
+                        func perform(with status: Bool) {
+                            //Lock/Unlock Cookie Folder
+                            let success = FileManager.default.lockFolders([
+                                (.cookie, status),
+                                (.webSiteData, status)
+                                ])
+                            if success {
+                                Preferences.Privacy.blockAllCookies.value = status
+                            } else {
+                                //Revert the changes. Not handling success here to avoid a loop.
+                                _ = FileManager.default.lockFolders([
+                                    (.cookie, !status),
+                                    (.webSiteData, !status)
+                                    ])
+                                self.undoToggleEvent(section: 1, row: 1)
+                                
+                                // TODO: Throw Alert to user to try again?
+                                let alert = UIAlertController(title: nil, message: Strings.Block_all_cookies_failed_alert_msg, preferredStyle: .alert)
+                                let okAction = UIAlertAction(title: Strings.OKString, style: .default)
+                                alert.addAction(okAction)
```

Comment:
okAction can just be created inline since its not used elsewhere and is only saving 1 character in length

Reply:
Done